### DISCUSSION TITLE
RC1180 Base Board

### DISCUSSION THREAD

**Discussion by jacek27** [wmbusmeters/wmbusmeters#564] [Show and tell]
Hi, 

I've prepared an USB adapter for board for a RC1180-MBUS3 modules.
This was dictated by scarce availability of the RC1180-MBUS3-USB dongle by RadioCraft.
 
If anybody would be interested in testing feel free to reach out for pcbs 

https://github.com/jacek27/rc1180_dongle

**Comment by weetmuts** [comment]
Very nice! Can you post a working example on the wmbusmeterswiki with photos etc?
